main.cpp

```diff
@@ -40,6 +41,11 @@ void GetDevice(void (*callback)(wgpu::Device)) {
             [](WGPURequestDeviceStatus status, WGPUDevice cDevice,
                const char* message, void* userdata) {
               wgpu::Device device = wgpu::Device::Acquire(cDevice);
+              device.SetUncapturedErrorCallback(
+                  [](WGPUErrorType type, const char* message, void* userdata) {
```

Comment:
Is there a way to use `wgpu::ErrorType` or does also https://bugs.chromium.org/p/dawn/issues/detail?id=1892 apply there?

Reply:
No, you have to use the C type because of that.